packages/amplify-category-function/src/__tests__/provider-utils/awscloudformation/utils/environmentVariablesHelper.test.ts

```diff
@@ -1,15 +1,13 @@
-import {
-  stateManager, JSONUtilities, $TSContext, pathManager,
-} from 'amplify-cli-core';
+import { stateManager, JSONUtilities, $TSContext, pathManager } from 'amplify-cli-core';
 import { prompter } from 'amplify-prompts';
-import {
-  ensureEnvironmentVariableValues,
-  getStoredEnvironmentVariables,
-  saveEnvironmentVariables,
-} from '../../../../provider-utils/awscloudformation/utils/environmentVariablesHelper';
+import * as m from '../../../../provider-utils/awscloudformation/utils/environmentVariablesHelper';
```

Comment:
I think we should change this variable name to something more informative because I was looking through the tests and I was confused about what `m` meant

Reply:
that makes sense to me. Will change